plutus-playground-client/src/Chain.purs

```diff
@@ -61,14 +65,15 @@ evaluationPane e@(EvaluationResult {emulatorLog, resultBlockchain}) =
                 [ class_ $ ClassName "logs" ]
                 (emulatorEventPane <$> Array.reverse logs)
         ]
-    , br_
-    , div_
-        [ h2_ [ text "Chain" ]
-        , slot' cpMockchainChart MockchainChartSlot
-            (echarts Nothing)
-            ({width: 930, height: 600} /\ unit)
-            (input HandleMockchainChartMessage)
-        ]
+    -- TODO Needs adapting for multicurrency.
+    -- , br_
+    -- , div_
+    --     [ h2_ [ text "Chain" ]
+    --     , slot' cpMockchainChart MockchainChartSlot
+    --         (echarts Nothing)
+    --         ({width: 930, height: 600} /\ unit)
+    --         (input HandleMockchainChartMessage)
+    --     ]
```

Comment:
what's happening with this?

Reply:
Disabled for the moment. I don't think it's worth delaying multi currency for, but it needs a fair bit of work to update it.